shoop/core/models/_shipments.py

```diff
@@ -45,6 +45,7 @@ class Shipment(models.Model):
     description = models.CharField(max_length=255, blank=True, verbose_name=_("description"))
     volume = MeasurementField(unit="m3", verbose_name=_("volume"))
     weight = MeasurementField(unit="kg", verbose_name=_("weight"))
+    shipment_key = models.CharField(max_length=64, unique=True, verbose_name=_("shipment key"))
```

Comment:
Order has key though. Hmm... but that's actually not the same usage either... yeah, identifier might be better for this.